fendermint/vm/interpreter/src/fvm/mod.rs

```diff
@@ -56,7 +57,10 @@ impl<C> ValidatorContext<C> {
 
 /// Interpreter working on already verified unsigned messages.
 #[derive(Clone)]
-pub struct FvmMessageInterpreter<DB, C> {
+pub struct FvmMessageInterpreter<DB, C>
+where
+    DB: fvm_ipld_blockstore::Blockstore + 'static + Clone,
```

Comment:
```suggestion
    DB: Blockstore + 'static + Clone,
```
Just a bit of a mouthful and usually it appears without the prefix.